nifi-nar-bundles/nifi-druid-bundle/nifi-druid-controller-service/src/main/java/org/apache/nifi/controller/druid/DruidTranquilityController.java

```diff
@@ -43,10 +43,7 @@
 import org.apache.nifi.annotation.documentation.Tags;
 import org.apache.nifi.annotation.lifecycle.OnDisabled;
 import org.apache.nifi.annotation.lifecycle.OnEnabled;
-import org.apache.nifi.components.AllowableValue;
-import org.apache.nifi.components.PropertyDescriptor;
-import org.apache.nifi.components.ValidationContext;
-import org.apache.nifi.components.ValidationResult;
+import org.apache.nifi.components.*;
```

Comment:
Travis is going to complain about that star import line when it runs `contrib-check`, it will show up as a CheckStyle violation. You can find these ahead of time by running with the `-Pcontrib-check` profile activated when you do your local builds (just for the bundle you're touching, don't worry about the whole thing -- Travis CI will do that)